libr/bin/dwarf.c

```diff
@@ -424,7 +424,9 @@ static const ut8 *parse_line_header (
 		if (f) {
 			fprintf (f, " Opcodes:\n");
 		}
-		for (int i = 1; i <= hdr->opcode_base - 1; i++) {
+		
+		int i;
```

Comment:
I already asked for this fix before merging this PR. actually this line had 3 bugs:

* use size_t instead of int
* change <= -1 with just <
* move the int variable outside that place

Reply:
It was fixed in that PR already.

Reply:
Didn't notice #17077 
I'll close this one.